Example/SmileID/HomeViewModel.swift

```diff
@@ -1,24 +1,40 @@
-import Foundation
 import Combine
-import UIKit
+import Foundation
+import Sentry
 import SmileID
+import SwiftUI
+import UIKit
 
 class HomeViewModel: ObservableObject,
     SmartSelfieResultDelegate,
     DocumentVerificationResultDelegate,
     EnhancedDocumentVerificationResultDelegate,
     EnhancedKycResultDelegate,
     BiometricKycResultDelegate {
-
     // MARK: - UI Properties
+
     @Published var dismissed = false
     @Published var toastMessage = ""
     @Published var showToast = false
     @Published var partnerId: String
+    @ObservedObject var networkMonitor = NetworkMonitor.shared
 
     init(config: Config) {
         partnerId = config.partnerId
         SmileID.initialize(config: config, useSandbox: true)
+        SentrySDK.configureScope { scope in
+            scope.setTag(value: "partner_id", key: self.partnerId)
+            let user = User()
+            user.email = self.partnerId
```

Comment:
This should be the id

```suggestion
            user.userId = self.partnerId
```